components/dialog/dialog-styles.js

```diff
@@ -94,6 +94,15 @@ export const dialogStyles = css`
 		height: 100%;
 	}
 
+	:host([critical]) .d2l-dialog-header {
+		border-block-start: 0.4rem solid var(--d2l-color-cinnabar);
+		border-start-start-radius: 0.4rem;
+		border-start-end-radius: 0.4rem;
+		margin-block: -1px 0;
+		margin-inline: -1px;
+		padding: 1rem 31px 23px 31px;
```

Comment:
I combined all the padding and put the 23px up here as well. I felt it was a bit more readable with the disadvantage that if the 23px changes in the regular `.d2l-dialog-header` css then it would have to get changed here as well.